### PR TITLE
[SW-119] Bullet switcher

### PR DESCRIPTION
**Pull request by pstachel**

### CODE REVIEW

**Review by ifaouibadi — APPROVED** [review]

File: src/popup/router/components/BulletSwitcher.vue [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+          v-for="(_, idx) in optionsSize"
+          :key="idx"
+          class="bullet"
+          :class="{
+            medium: getMediumCondition(idx),
+            big: getBigCondition(idx),
+            active: idx === currentIdx,
+          }"
+          :style="getBulletTranslation(idx)"
+          @click="() => $emit('change', idx)"
```
Comment (CedrikNikita) [inline src/popup/router/components/BulletSwitcher.vue:18]:
```suggestion
          @click="$emit('change', idx)"
```
Why shouldn't we do it like that?

File: src/popup/router/components/BulletSwitcher.vue [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        />
+      </div>
+    </div>
+    <PlusCircle
+      :class="{ active: currentIdx === optionsSize }"
+      @click="() => $emit('change', optionsSize)"
```
Comment (CedrikNikita) [inline src/popup/router/components/BulletSwitcher.vue:24]:
```suggestion
      @click="$emit('change', optionsSize)"
```

**Comment by Liubov-crypto** [comment]
As I see there are different fonts, and the account address in the wallet is longer than in Figma. Also in the wallet, the send/receive buttons are white, and in Figma they are transparent.
@pstachel  @ifaouibadi  please check it
figma : wall

File: src/styles/variables.scss [hunk 3]
```diff
@@ -7,13 +7,15 @@ $font-mono: 'IBM Plex Mono';
 
 $color-white: #fff;
 $color-black: #000;
+$color-purple: #3270ff;
```
Comment (peronczyk) [inline src/styles/variables.scss:10]: This variable is not used anymore.

**Review by ifaouibadi — APPROVED** [review]

**Pull request merged**